doc/guide/admin_slurm.rst

```diff
@@ -0,0 +1,66 @@
+.. _admin-guide:
+
+********************
+Migrating from Slurm
+********************
+
+Flux Maturity
+=============
+
+*I run a large HPC center.  Should I kick Slurm to the curb?*
+
+Not today.  Enjoy your Slurm (it's highly addictive!) and check back
+on Flux at the end of 2026.
+
+TL;DR
+-----
+
+Flux is in daily production use as the system workload manager on El Capitan
+(currently in slot 1 of the TOP500) and its unclassified sister systems at
+LLNL.  The process of standing up those systems brought the Flux team together
+to address problems of scale and stability with unprecedented urgency.
+As a result, system deployments of Flux on virtually any size system are
+viable at this point.  That said, the process also brought into focus
+missing features, now prioritized.  As of December 2025:
+
+- Binary packages are only published internally for the TOSS operating
+  system.  Source RPM packages for RHEL 8 and 9 are manually attached to
+  github releases.  Wider operating system support and binary package
+  distribution are being discussed.
+
+- All nodes of Flux instance must run the same flux broker release version.
+  When flux-core 1.0.0 is released, relaxed version interoperability rules
+  will be published.  For now, rolling upgrades are not possible.
+
+- Support for restarting Flux without perturbing running jobs is expected
+  late 2026.
+
+- Reservation support is expected mid 2026.
+
+- Flux's flexible resource representation is not yet mature.  Meanwhile,
+  hardware locality and binding needs of modern applications can be tricky
+  to satisfy in Flux.
+
+- The system Flux does not preserve job :term:`step` data by default.
+
+- No centralized accounting database.
```

Comment:
I'm just curious, what is meant by this?  Like `flux-accounting` working across multiple clusters vs a single one?

Could mention what `flux-accounting` can do instead?